types/Building.ts

```diff
@@ -0,0 +1,34 @@
+import { Building, BuildingEntry } from '@prisma/client';
+
+export type BuildingDO = Building & { building_entries: BuildingEntry[] };
+
+export type BuildingEntryDTO = {
+  id: BuildingEntry['building_entry_id'];
+  longitude: BuildingEntry['map_longitude'];
+  latitude: BuildingEntry['map_latitude'];
+  createdAt: BuildingEntry['created_at'];
+  updatedAt: BuildingEntry['updated_at'];
+};
+export const buildingEntryDOtoDTO = (entry: BuildingEntry): BuildingEntryDTO => ({
```

Comment:
Define `export type BuildingEntryDO = BuildingEntry`, even if the new type is equal to the base type